tests/Stuart/tests/ClientTest.php

```diff
@@ -144,4 +145,18 @@ public function test_validate_a_dropoff_address()
         self::assertEquals($validity->success, true);
         \Phake::verify($this->httpClient)->performGet('/v2/addresses/validate', $query);
     }
+
```

Comment:
it should contains two tests: `test_get_scheduling_slots_at_pickup` and `test_get_scheduling_slots_at_dropoff`